### ISSUE TITLE
Crash

### ISSUE THREAD

**Issue by Ya-Block** [djoslin0/sm64ex-coop#253]
wanted to join a public server and i got a crash
![image](https://github.com/djoslin0/sm64ex-coop/assets/146646463/4032fb97-d248-4c3e-9428-7877dbee633f)

**Comment by yoyeet961** [comment]
does it happen all the time? if not then this is not a reason to make a github issue

**Comment by TimeParadoxYT** [comment]
yeah, crashes are bound to happen. If it doesn't happen often its nothing to worry about

**Comment by Ya-Block** [comment]
it only happened once

**Comment by yoyeet961** [comment]
Then close this issue. Crashes are really common and won't cause any harm.
